source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +487,380 @@ These events apply only for connections using the ``connection API``.
+Mail User
+=========
 
-.. _event_mailbox:
+.. dovecot_core:event:: mail_user_session_finished
+   :added: v2.3.19
+   :inherit: mail_user
 
-Mailbox
--------
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_storage`                                         |
-+---------------------+------------------------------------------------------+
-| mailbox             | Full mailbox name in UTF-8                           |
-|                     |                                                      |
-|                     | .. versionadded:: v2.3.9                             |
-+---------------------+------------------------------------------------------+
-| mailbox_guid        | Mailbox GUID with obox storage                       |
-|                     |                                                      |
-|                     | .. versionadded:: v2.3.10                            |
-+---------------------+------------------------------------------------------+
+   :field utime: User CPU time used in microseconds
+   :field stime: System CPU time used in microseconds
+   :field minor_faults: Page reclaims (soft page faults)
+   :field major_faults: Page faults (hard page faults)
+   :field vol_cs: Voluntary context switches
+   :field invol_cs: Involuntary context switches
+   :field rss: Resident set size in bytes.
+               (Skipped in non-Linux environments.)
+   :field vsz: Virtual memory size in bytes.
+               (Skipped in non-Linux environments.)
+   :field rchar: I/O counter: chars (bytes) read from storage
+                 (Skipped in non-Linux environments.)
+   :field wchar: I/O counter: chars (bytes) written to storage
+                 (Skipped in non-Linux environments.)
+   :field syscr: Number of read syscalls
+                 (Skipped in non-Linux environments.)
+   :field syscw: Number of write syscalls
+                 (Skipped in non-Linux environments.)
 
-.. _event_mail_expunged:
 
-mail_expunged
-^^^^^^^^^^^^^
+Mailbox
+-------
 
-.. versionadded:: 2.3.15
+.. dovecot_core:event:: mail_expunged
+   :added: v2.3.15
+   :inherit: mailbox
 
-A mail was expunged from the mailbox. Note that this event inherits from
-mailbox, not mail.
+   :field uid: UID of the expunged mail.
 
-+---------------------+--------------------------------------------------------+
-| Field               | Description                                            |
-+=====================+========================================================+
-| uid                 | UID of the expunged mail.                              |
-+---------------------+--------------------------------------------------------+
+   A mail was expunged from the mailbox. Note that this event inherits from
+   mailbox, not mail.
```

Comment:
mailbox & mail could be links?

Reply:
See above: there is nothing to link to, as the inherited group is meant for internal use only (it's not a link on the page).